optimum/neuron/utils/misc.py

```diff
@@ -45,7 +45,7 @@
 logger = logging.get_logger()
 
 
-def should_log() -> bool:
+def should_current_worker_log() -> bool:
```

Comment:
It still has nothing to do with log. I would call it is_main_worker, wdyt ?

Reply:
Alright, doing it.